plugins/main/bridge/bridge.go

```diff
@@ -589,6 +655,39 @@ func cmdDel(args *skel.CmdArgs) error {
 		}
 	}
 
+	// Remove bride if prunening is required
+	if n.Prune == true {
+		links, err := netlink.LinkList()
+		if err != nil {
+			return err
+		}
+
+		var br netlink.Link
+		brIndex := 0
+		for _, l := range links {
+			if l.Attrs().Name == n.BrName {
+				brIndex = l.Attrs().Index
+				br = l
+				break
+			}
+		}
+
+		if brIndex == 0 {
+			return fmt.Errorf("cannot found bridge")
+		}
+
+		portCount := 0
```

Comment:
As there is used break after first found interface - keeping that as `int` instead of `bool` (lets say: `hasSlaves`) has less sense now.